plugins/aws/elasticache/elasticacheDefaultPorts.spec.js

```diff
@@ -0,0 +1,161 @@
+const expect = require('chai').expect;
+var elasticacheDefaultPort = require('./elasticacheDefaultPorts');
+
+const describeClusters = [
+    {
+        "CacheClusterId": "test-dev",
+        "ConfigurationEndpoint": {
+            "Address": "test-dev.1234.aaa.use1.cache.amazonaws.com",
+            "Port": 11111
+        },
+        "ClientDownloadLandingPage": "https://console.aws.amazon.com/elasticache/home#client-download:",
+        "CacheNodeType": "cache.r6g.large",
+        "Engine": "memcached",
+        "EngineVersion": "1.6.6",
+        "CacheClusterStatus": "available",
+        "NumCacheNodes": 1,
+        "PreferredAvailabilityZone": "us-east-1b",
+        "CacheClusterCreateTime": "2021-08-23T17:26:45.535Z",
+        "PreferredMaintenanceWindow": "mon:09:30-mon:10:30",
+        "PendingModifiedValues": {},
+        "CacheSecurityGroups": [],
+        "CacheParameterGroup": {
+            "CacheParameterGroupName": "default.memcached1.6",
+            "ParameterApplyStatus": "in-sync",
+            "CacheNodeIdsToReboot": []
+        },
+        "CacheSubnetGroupName": "test-dev-subnet-group",
+        "AutoMinorVersionUpgrade": true,
+        "SecurityGroups": [
+            {
+                "SecurityGroupId": "sg-1234",
+                "Status": "active"
+            }
+        ],
+        "AuthTokenEnabled": false,
+        "TransitEncryptionEnabled": false,
+        "AtRestEncryptionEnabled": false,
+        "ARN": "arn:aws:elasticache:us-east-1:978540712345:cluster:test-dev"
+    },
+    {
+        "CacheClusterId": "test-dev",
+        "ConfigurationEndpoint": {
+            "Address": "test-dev.1234.aaa.use1.cache.amazonaws.com",
+            "Port": 11211
+        },
+        "ClientDownloadLandingPage": "https://console.aws.amazon.com/elasticache/home#client-download:",
+        "CacheNodeType": "cache.r6g.large",
+        "Engine": "memcached",
+        "EngineVersion": "1.6.6",
+        "CacheClusterStatus": "available",
+        "NumCacheNodes": 1,
+        "PreferredAvailabilityZone": "us-east-1b",
+        "CacheClusterCreateTime": "2021-08-23T17:26:45.535Z",
+        "PreferredMaintenanceWindow": "mon:09:30-mon:10:30",
+        "PendingModifiedValues": {},
+        "CacheSecurityGroups": [],
+        "CacheParameterGroup": {
+            "CacheParameterGroupName": "default.memcached1.6",
+            "ParameterApplyStatus": "in-sync",
+            "CacheNodeIdsToReboot": []
+        },
+        "CacheSubnetGroupName": "test-dev-subnet-group",
+        "AutoMinorVersionUpgrade": true,
+        "SecurityGroups": [
+            {
+                "SecurityGroupId": "sg-1234",
+                "Status": "active"
+            }
+        ],
+        "AuthTokenEnabled": false,
+        "TransitEncryptionEnabled": false,
+        "AtRestEncryptionEnabled": false,
+        "ARN": "arn:aws:elasticache:us-east-1:978540712345:cluster:test-dev"
```

Comment:
We should not be displaying account if
```suggestion
        "ARN": "arn:aws:elasticache:us-east-1:111122223333:cluster:test-dev"
```